data/Han/data.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "HanChain",
+    "symbol": "HAN",
+    "decimals": 18,
+    "website": "https://paykhan.io",
+    "telegram": "https://t.me/hanchain_official",
+    "twitter": "https://twitter.com/HanIdentity",
+    "discord": "https://discord.gg/5gtfUuvJJX",
+    "medium": "https://medium.com/@HanIdentity",
+    "youtube": "https://www.youtube.com/channel/UCQPzdwU4KHlXO3srolte0Dg",
+    "facebook": "https://www.facebook.com/HanChainGlobalOfficial-101331419212206",
```

Comment:
Currently only the fields "website" and "twitter" are accepted -- "telegram", "discord", "medium", and "facebook" are not. Please remove these fields and validation will pass.

Reply:
remove fields thx